tests/test_codecs.py

```diff
@@ -670,6 +670,11 @@ async def test_unhandled_type_fallback(self):
             ''')
 
     async def test_invalid_input(self):
+        # The latter message appears beginning in Python 3.10.
+        integer_required = (
+            "(an integer is required|"
+            "\('str' object cannot be interpreted as an integer\))")
```

Comment:
```suggestion
            r"(an integer is required|"
            r"\('str' object cannot be interpreted as an integer\))")
```

Reply:
Thanks, I agree with that change. I’ve been switching languages too much lately, or I would have remembered to do it that way to begin with.